test/fixtures/mui/.babelrc

```diff
@@ -0,0 +1,19 @@
+{
+  "plugins": [
+    [
+      "../../../src",
+      {
+        "fileName": false,
+        "transpileTemplateLiterals": false,
+        "ssr": true,
+        "topLevelImportPaths": [
+          "@mui/material",
+          "@mui/material/*",
```

Comment:
FWIW at HEAD right now this doesn't work. You need a regex. I have a PR to change this to picomatch